integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt

```diff
@@ -5,7 +5,7 @@ internal object TestedVersions {
     val BASE =
         BuildVersions.permutations(
             gradleVersions = listOf("7.3", "6.9"),
```

Comment:
The [latest](https://gradle.org/releases/) Gradlet version is 7.4.2

Reply:
Indeed, might as well update it and AGP :+1: